src/Main.js

```diff
@@ -1,31 +1,72 @@
-import { Component } from 'react';
-
-// main needs to know the title pass in... with props
+import {Component} from 'react';
+import Col from 'react-bootstrap/Col';
+import Container from 'react-bootstrap/Container';
+import Image from 'react-bootstrap/Image';
+import Row from 'react-bootstrap/Row';
 
 class Main extends Component {
   render() {
+    const bios = this.props.beastBios;
+
     return (
-      <>
-      <div>
-        <h2>I am main. My message is {this.props.message}</h2>
-        </div>
-        <HornedBeast title={'Rhino'} img={<img src="https://images.unsplash.com/photo-1512636618879-bbe79107e9e3?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=bd9460ee6d1ddbb6b1ca7be86dfc4590&auto=format&fit=crop&w=1825&q=80" alt="Rhino Family" title="Rhino Family"></img>} desc={<p>I have one giant horn.</p>}/>
-        <HornedBeast title={'Unicorn'} img={<img src="https://www.dhresource.com/0x0s/f2-albu-g5-M00-1A-11-rBVaI1hsIIiALxKzAAIHjSU3VkE490.jpg/wholesale-halloween-costume-prop-unicorn.jpg" alt="Unicorn Head" title="Unicorn Head"></img>} desc={<p> I also have one giant horn. </p>}/>
-      </>
-    )
+      <Container>
+        <h2>{this.props.message}</h2>
+        <Row>
+          <Col><HornedBeastImg bio={bios[0]} /></Col>
+          <Col><HornedBeastImg bio={bios[1]} /></Col>
+          <Col><HornedBeastImg bio={bios[2]} /></Col>
+          <Col><HornedBeastImg bio={bios[3]} /></Col>
```

Comment:
Really lovely work so far! 👍 

I am sure you are workin on lab 2, so I would clean this block up a bit spacing-wise.